packages/cdk/lib/roles/policies/cromwell-batch-policy.ts

```diff
@@ -0,0 +1,33 @@
+import * as iam from "monocdk/aws-iam";
+
+export interface CromwellBatchPolicyProps {
+  account: string;
+  region: string;
+  jobQueueArn: string;
+}
+
+export class CromwellBatchPolicy extends iam.PolicyDocument {
+  constructor(props: CromwellBatchPolicyProps) {
+    const cromwellJobArn = `arn:aws:batch:${props.region}:${props.account}:job-definition/cromwell_*`;
+    super({
+      assignSids: true,
+      statements: [
+        new iam.PolicyStatement({
+          effect: iam.Effect.ALLOW,
+          actions: ["batch:DescribeJobDefinitions", "batch:ListJobs", "batch:DescribeJobs", "batch:DescribeJobQueues", "batch:DescribeComputeEnvironments"],
+          resources: ["*"],
```

Comment:
Is it possible to scope these down to a specific job queue where possible? Same for the nextflow batch policy

https://docs.aws.amazon.com/batch/latest/userguide/ExamplePolicies_BATCH.html#iam-example-restrict-job-queue

Reply:
Unfortuently not, these are all account level permissions: https://docs.aws.amazon.com/service-authorization/latest/reference/list_awsbatch.html